### PR TITLE
404 page always rendered using JS components

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
Remove twirl template for 404 page

## What is the value of this and can you measure success?
1 page (404) fully served using the new javascript rendring layout

## Does this affect other platforms - Amp, Apps, etc?
No

## Tested in CODE?
No

### CODE REVIEW

File: common/app/mvt/MultiVariateTesting.scala [hunk 1]
```diff
@@ -56,7 +56,7 @@ object ABJavascriptRenderingVariant extends TestDefinition(
   name = "ab-javascript-rendering-variant",
   description = "Users in this test will see pages rendered by the javascript renderer.",
   owners = Seq(Owner.withName("dotcom.platform")),
-  sellByDate = new LocalDate(2017, 8, 29)
+  sellByDate = new LocalDate(2017, 9, 27)
```
Comment (TBonnin) [inline common/app/mvt/MultiVariateTesting.scala]: Keeping the switch for now. I feel we are going to need it again very soon 🐎

**Comment by sndrs** [comment]
oooohhww!!!

just noticed we might want to add ophan to the LS page before merging this...

**Comment by TBonnin** [comment]
@sndrs Ok. Feel free to merge this PR once adding ophan is done

File: archive/app/views/notFound.scala.html [hunk 2]
```diff
@@ -1,128 +0,0 @@
...
-            ga('set', 'dimension14', '404');
-            ga('send', 'pageview');
-        </script>
-
-        <script src="https://pasteup.guim.co.uk/js/lib/requirejs/2.1.5/require.min.js"></script>
-
-        <script>
-            require.config({
-                paths: {
-                    'ophan/http-status' : '//j.ophan.co.uk/ophan.http-status',
```
Comment (sndrs) [inline archive/app/views/notFound.scala.html]: @SiAdcock do you recognise this code? can we load this as a single file for now, or do we need to install/bundle ophan?

**Review by NataliaLKB — APPROVED** [review]
lgtm 👍 

Looking forward to seeing any results and next stages!

**Comment by sndrs** [comment]
@TBonnin I've addd the tracking back, but i cannot get the page to run, i get `Uncaught error from thread [application-akka.actor.default-dispatcher-5] shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[application]` and SBT shuts down.

My guess is it's a JS error but I cannot trace it at the moment because of that...

(added DNM flag as a result, not because of anything else)

**Comment by TBonnin** [comment]
@sndrs I am looking into that

**Comment by TBonnin** [comment]
@sndrs 
I cannot reproduce your issue. 404 page renders properly locally with the ophan related script page. 
I also deployed it on CODE and it is also working fine
screen shot 2017-08-16 at 10 06 25

**Comment by jfsoul** [comment]
@TBonnin what's with the `@{Configuration.debug.beaconUrl}` in that screeny? 👀

File: ui/src/views/404/index.jsx [hunk 3]
```diff
@@ -42,4 +42,42 @@ export default () =>
...
+                    });
+                    require(['ophan/http-status'], function(reporter) {
+                    reporter.reportStatus('next-gen', 404);
+                    });
+                `,
+            }}
+        />
+
+        <img
+            src="@{Configuration.debug.beaconUrl}/count/40x.gif"
```
Comment (NataliaLKB) [inline ui/src/views/404/index.jsx]: @jfsoul found this. Looks like this has been copy pasted from a twirl template

**Comment by NataliaLKB** [comment]
Great spot @jfsoul !

**Comment by TBonnin** [comment]
@jfsoul good catch. Will check with @sndrs and fix

**Comment by sndrs** [comment]
that little image needs some config for it actually – don't think we have a way of getting that in yet, want to maybe add a `tracking.beaconUrl` to a state object @TBonnin? 

my problem was fixed by cleaning sbt (ta @NataliaLKB) so that's all good

**Comment by sndrs** [comment]
removed DNM tag, as it was my SBT not the PR

**Comment by TBonnin** [comment]
@sndrs @NataliaLKB @jfsoul 
Now passing the config value to the component 25d4bb4

File: archive/app/ui/NotFound.scala [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
 
+import conf.Configuration
+import play.api.libs.json.{JsValue, Json}
 import rendering.Renderable
 
 object NotFound extends Renderable {
-  override def props = None
+  override def props: Option[JsValue] = Some(
+    Json.obj(
+      "beaconUrl" -> Configuration.debug.beaconUrl
```
Comment (TBonnin) [inline archive/app/ui/NotFound.scala]: We will need to agree on a props model when components are going to become more complicated and routing will be required. @sndrs

Comment (sndrs) [inline archive/app/ui/NotFound.scala]: yeah this looks good to me for now. i guess that will almost design itself as we go along?

Comment (NataliaLKB) [inline archive/app/ui/NotFound.scala]: Maybe something like:

```
{
  component: "/404",
  configuration: {
    beaconUrl: "myUrl"
  }
}
```

With the thought that later we might want:

```
{
  component: "/404",
  configuration: {
    ...
  },
  page: {
    ...
  }
}
```

Comment (TBonnin) [inline archive/app/ui/NotFound.scala]: @NataliaLKB totally.
Let's seat down an make a decision about this

**Pull request merged**